sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallMediaAsync.java

```diff
@@ -314,4 +323,28 @@ private PlaySourceInternal translatePlaySourceToPlaySourceInternal(PlaySource pl
         }
         return playSourceInternal;
     }
+
+    private List<RecognizeChoice> translateRecognizeChoiceListtoInternal(List<com.azure.communication.callautomation.models.RecognizeChoice> recognizeChoices) {
+        return recognizeChoices.stream()
+            .map(e -> translateRecognizeChoiceToInternal(e))
```

Comment:
I bet this could be changed to `.map(this::translateRecognizeChoiceToInternal)`
check the `Readme` file in `swagger` folder for ideas on how to do it for models and enums.

Reply:
good point, will change the style